src/robocup_client/message_handler/message_handler.cpp

```diff
@@ -36,7 +39,7 @@ void MessageHandler::add_motor_position_in_degree(std::string name, double posit
 {
   auto motor_position = actuator_request->add_motor_positions();
   motor_position->set_name(name);
-  motor_position->set_position(keisan::deg_to_rad(position));
+  motor_position->set_position(position * M_PI / 180.0);
```

Comment:
(just asking) why don't you use `keisan`'s one?

Reply:
Done